### PR TITLE
Mockbackend

### PR DESCRIPTION
**Pull request by bcapdevilac**

### CODE REVIEW

File: src/components/navigation/navigation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  it('navigation items', inject([Navigation, MockBackend], (navigation, backend) => {
+    backend.connections.subscribe(c => {
+      expect(c.request.url).toMatch('@components/navigation');
+      var response = [
+        {
+          "data": {
+            "items": [
+              {
+                "title":"Home",
+                "url":"http://castanyera.iskra.cat:8070/Plone"
```
Comment (tisto) [inline src/components/navigation/navigation.spec.ts]: @bcapdevilac I'm wondering if we should use "http://localhost:8080/Plone" here as base URL to be consistent with the other responses in the tests.

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 language: node_js
```
Comment (tisto) [inline .travis.yml:1]: @bcapdevilac shouldn't we run at least the build on travis for now to make sure that works?

File: package.json [hunk 3]
```diff
@@ -79,15 +69,14 @@
...
-    "angular2-universal": "~0.101.4",
+    "@angular/common": "2.0.0-rc.2",
+    "@angular/compiler": "2.0.0-rc.2",
+    "@angular/core": "2.0.0-rc.2",
+    "@angular/http": "2.0.0-rc.2",
+    "@angular/platform-browser": "2.0.0-rc.2",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.2",
+    "@angular/platform-server": "2.0.0-rc.2",
+    "@angular/router": "3.0.0-alpha.6",
     "body-parser": "^1.15.0",
```
Comment (tisto) [inline package.json:80]: @ebrehault @bloodbare I'm wondering if we should specifically pin (no "^" or "~") all versions right from the start to have a more stable CI build. What do you usually do in your JS projects?

**Comment by tisto** [comment]
@bcapdevilac I can start the server when I run "npm install" and "npm start". Though, I see an error message in the log:

```
[default] /Users/timo/workspace/plone/plone_client/src/components/views/chooser.component.ts:47:49
    Property '_platformLocation' does not exist on type 'LocationStrategy'
```

Running "npm test" works, so I guess we could just build and test on travis, right?

When I open the app in the browser I see a few js errors:

```
EXCEPTION: Error: Uncaught (in promise): TypeError: Cannot convert undefined or null to object
zone.js:461Unhandled Promise rejection: Cannot convert undefined or null to object ; Zone: angular ; Task: Promise.then ; Value: TypeError: Cannot convert undefined or null to object(…)consoleError @ zone.js:461
zone.js:463Error: Uncaught (in promise): TypeError: Cannot convert undefined or null to object(…)

```

File: package.json (hunk 3 above)
Comment (ebrehault) [inline package.json:80]: @tisto yes we should do that

File: src/components/navigation/navigation.spec.ts (hunk 1 above)
Comment (bcapdevilac) [inline src/components/navigation/navigation.spec.ts]: Yes, your're right! I've already changed it

File: .travis.yml (hunk 2 above)
Comment (bloodbare) [inline .travis.yml:1]: @tisto its already active on travis-ci

**Comment by bcapdevilac** [comment]
@tisto about the error log you're talking about:

```
[default] /Users/timo/workspace/plone/plone_client/src/components/views/chooser.component.ts:47:49
    Property '_platformLocation' does not exist on type 'LocationStrategy'
```

 You have to have node > 5

**Pull request merged**